test/common/stats/stat_merger_fuzz_test.cc

```diff
@@ -40,6 +40,9 @@ void testDynamicEncoding(absl::string_view data, SymbolTable& symbol_table) {
     // segments, which trigger some inconsistent handling as described in that
     // bug.
     uint32_t num_bytes = (1 + data[index]) & 0x7;
+    if (index == 0 && num_bytes == 0) {
```

Comment:
There seems to be an inconsistency between the line above and the comment before it.  The code above generate num_bytes between 0 and 7, not 1 and 8.

https://github.com/envoyproxy/envoy/pull/10965/files#r629769284

Reply:
I think you are right; the code should be changed to reflect the comment. However it's not material to the problems in this fuzz test.

There's an outstanding PR that will fix it, however: https://github.com/envoyproxy/envoy/pull/16239

@jessicayuen will you be able to push that forward?

I think the fix offered here is too aggressive; it doesn't just skip over the troublesome cases but completely stops the test on the passed-in corpus. It could just 'continue'.